react/src/components/StorageStatusPanelCard.tsx

```diff
@@ -0,0 +1,125 @@
+import { useSuspendedBackendaiClient } from '../hooks';
+import { useSuspenseTanQuery } from '../hooks/reactQueryAlias';
+import { useCurrentProjectValue } from '../hooks/useCurrentProject';
+import BAICard, { BAICardProps } from './BAICard';
+import BAIPanelItem from './BAIPanelItem';
+import { StorageStatusPanelCardQuery } from './__generated__/StorageStatusPanelCardQuery.graphql';
+import { Col, Row, theme } from 'antd';
+import graphql from 'babel-plugin-relay/macro';
+import _ from 'lodash';
+import React, { useDeferredValue } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLazyLoadQuery } from 'react-relay';
+
+interface StorageStatusPanelProps extends BAICardProps {
+  fetchKey?: string;
+}
+
+const StorageStatusPanelCard: React.FC<StorageStatusPanelProps> = ({
+  fetchKey,
+  ...cardProps
+}) => {
+  const { t } = useTranslation();
+  const { token } = theme.useToken();
+  const baiClient = useSuspendedBackendaiClient();
+  const currentProject = useCurrentProjectValue();
+  const deferredFetchKey = useDeferredValue(fetchKey);
+
+  const isExcludedCount = (status: string) => {
+    return _.includes(
+      ['delete-ongoing', 'delete-complete', 'delete-error'],
+      status,
+    );
+  };
+
+  const { data: vfolders } = useSuspenseTanQuery({
+    queryKey: ['vfolders', { deferredFetchKey }],
+    queryFn: () => {
+      return baiClient.vfolder.list(currentProject?.id);
+    },
+  });
+  const createdCount = vfolders?.filter(
+    (item: any) =>
+      item.is_owner &&
+      item.ownership_type === 'user' &&
+      !isExcludedCount(item.status),
+  ).length;
+  const projectCount = vfolders?.filter(
+    (item: any) =>
+      item.ownership_type === 'group' && !isExcludedCount(item.status),
+  ).length;
+  const invitedCount = vfolders?.filter(
+    (item: any) =>
+      !item.is_owner &&
+      item.ownership_type === 'user' &&
+      !isExcludedCount(item.status),
+  ).length;
```

Comment:
Is not possible to fetch count using `vfolder_nodes` with `filter`? Please check this https://github.com/lablup/backend.ai-webui/blob/88f1c2bed2cc0164891ded0068e7e33159419af5/react/src/pages/ComputeSessionListPage.tsx#L137-L152

Reply:
The StorageStatusPanelCard component will be used later in the VfolderNodeListPage, soI will modify it to use the fragment from 3232PR.

Reply:
I don't think it's possible to get it with filter at the moment: the is_owner information is not included in the vfolder_node.